x/concentrated-liquidity/pool_test.go

```diff
@@ -55,10 +55,9 @@ func (s *KeeperTestSuite) TestCalcOutAmtGivenIn() {
 			tokenOutDenom: "eth",
 			priceLimit:    sdk.NewDec(5004),
 			// we expect to put 42 usdc in and in return get .008398 eth back
-			// due to limited liquidity, we actually put in 41.99 usdc and in return get .008396 eth back
-			expectedTokenIn:  sdk.NewCoin("usdc", sdk.NewInt(41999999)),
-			expectedTokenOut: sdk.NewCoin("eth", sdk.NewInt(8396)),
-			expectedTick:     sdk.NewInt(85184),
+			expectedTokenIn:  sdk.NewCoin("usdc", sdk.NewInt(41999999)), // TODO: should be 42
+			expectedTokenOut: sdk.NewCoin("eth", sdk.NewInt(8396)),      // TODO: should be 0.008371593947078467
+			expectedTick:     sdk.NewInt(85184),                         // TODO: should be 85183
```

Comment:
Did we ever get to a wolframalpha for this or is this something that we should do in the future as well?

Reply:
The pool tests might be a little harder to replicate in wolfram because of the iterations we go through via the currently existing positions. There may be a simple way I am not seeing though